README.md

```diff
@@ -22,26 +22,33 @@ ___
 ___
 
 ## Introduction
+
 Anomalib is a deep learning library that aims to collect state-of-the-art anomaly detection algorithms for benchmarking on both public and private datasets. Anomalib provides several ready-to-use implementations of anomaly detection algorithms described in the recent literature, as well as a set of tools that facilitate the development and implementation of custom models. The library has a strong focus on image-based anomaly detection, where the goal of the algorithm is to identify anomalous images, or anomalous pixel regions within images in a dataset. Anomalib is constantly updated with new algorithms and training/inference extensions, so keep checking!
 
-##### Key features:
+**Key features:**
+
 - The largest public collection of ready-to-use deep learning anomaly detection algorithms and benchmark datasets.
 - [**PyTorch Lightning**](https://www.pytorchlightning.ai/) based model implementations to reduce boilerplate code and limit the implementation efforts to the bare essentials.
 - All models can be exported to [**OpenVINO**](https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/overview.html) Intermediate Representation (IR) for accelerated inference on intel hardware.
 - A set of [inference tools](#inference) for quick and easy deployment of the standard or custom anomaly detection models.
 
 ___
+
 ## Getting Started
+
 The repo is thoroughly tested based on the following configuration.
-*  Ubuntu 20.04
-*  NVIDIA GeForce RTX 3090
+
+- Ubuntu 20.04
+
+- NVIDIA GeForce RTX 3090
 
 You will need [Anaconda](https://www.anaconda.com/products/individual) installed on your system before proceeding with the Anomaly Library install.
 
 After downloading the Anomaly Library, extract the files and navigate to the extracted location.
 
 To perform an installation, run the following:
-```
+
+```bash
 yes | conda create -n anomalib python=3.8
 conda activate anomalib
 pip install -r requirements/base.txt
```

Comment:
Maybe we could also modify this bit. We could have an Installation section, one for pypi and one for editable install. Something like,

# Installation
## PyPI Install
```bash
pip install anomalib
```

## Local Install
For those who would like to edit,
```bash
git clone https://github.com/openvinotoolkit/anomalib.git
cd anomalib
pip install -e .
```

This is of course, quite raw an draft, we could explain it better

Reply:
Of course the virtualenv stuff could go under the local install. What do you think?

Reply:
I agree